Website/index.js

```diff
@@ -0,0 +1,474 @@
+
+        }
+    }
+        counter += 1
+    })
+
+    localStorage.setItem("name", String($("#name").val()));
+    $("<div id='date2' class='text-center'><h4>Hello " + String($("#name").val()) + "!</h4><p>Here is your duty for the week you selected</p></div>").insertBefore("#date");
+    $("#personalised").show();
+        $(["#date", "#date2", "#day_1", "#day_2", "#day_3", "#day_4", "#day_5"]).each(function (i,v) {$(v).hide()});
+        var eT = 0;
+    $.each(["#date2", "#day_1", "#day_2", "#day_3", "#day_4", "#day_5"], function (i, v) {
+        $(v).delay(eT).fadeIn("slow");
+        eT += 200;
+    });
+
+}
+
+function nameChecker() {
+    if ($("#name").val() === "Overview") {
+        localStorage.setItem("name", String($("#name").val()));
+        $("#personalised").hide();
+        $("#date2").remove();
+
+        
+        $.each(["#weekend", "#recess_duty_text", "#first_recess", "#second_recess", "#third_recess", "#fourth_recess", "#morning_duty_text", "#morning_duty_block"], function (i, v) {
+            $(v).hide();
+        });
+        $("#overview").show();
+        $("#start").attr("onchange", "update()");
+        update();
+    } else {
+        $("#overview").hide();
+        $("#start").attr("onchange", "show_personalised()")
+        show_personalised();
+    }
+}
+
+$.each(["#weekend", "#date", "overview"], function (i, v) {
+    $(v).hide();
+});
+
+$.getJSON("Data/data.json?time=" + new Date().getTime(),
+    function (data) {
+        var dt = new Date();
+        const monthFixer = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"];
+
+        var date = String(dt.getFullYear()) + "-" + String(monthFixer[dt.getMonth()]) + "-" + String(dt.getDate());
+        $("#start").attr("min", Object.keys(data["dates"])[0]);
+        $("#start").attr("max", Object.keys(data["dates"])[Object.keys(data["dates"]).length - 1]);
+
+        function formatDate(date) {
+        var d = new Date(date),
+            month = "" + (d.getMonth() + 1),
+            day = "" + d.getDate(),
+            year = d.getFullYear();
+
+        if (month.length < 2) {
+            month = "0" + month;
+        }
+        if (day.length < 2) {
+            day = "0" + day;
+        }
+
+        return [year, month, day].join('-');
+        }
+        $("#start").val(formatDate(dt));
+
+        jsonData = data;
+        getFlagsData();
+
+    }
+);
+
+function getFlagsData() {
+
+    $.getJSON("Data/flag.json?time=" + new Date().getTime(),
+        function (data) {
+            flagData = data;
+            getNameData();
+        }
+    );
+}
+
+function getNameData() {
+
+    $.getJSON("Data/names.json?time=" + new Date().getTime(),
+        function (data) {
+            nameData = data;
+
+            $.each(Object.keys(nameData), function (i, v) {
+                
+            if (localStorage.getItem("name") != null && localStorage.getItem("name") == v) {
+                $('option:selected', name_og).remove();
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
                $("option:selected", name_og).remove();
```